netflix/src/components/Genres.jsx

```diff
@@ -0,0 +1,16 @@
+// No actual logic here. Just some checkboxes.
+import React from "react";
+
+const Genres = () => {
+  const genres = ["Horror", "Comedy", "Documentary", "Classics", "Action"];
```

Comment:
Not a bug.

For your real-world applications, I suggest you to create key-values pairs to store option lists.
That is due to many reasons:
 - translations(you may want to render different name per selected language)
 - spaces/special characters in option names (generally, developers who need to work with HTTP hate this sort of stuff, as encoding/decoding those made many people insane)

So in real worlds apps, preferred way to have something like
```
const GENRES = {
   horror: "Horror",
   comedy: "Comedy"
};
```

So if you need to get list of values, you do something like `Object.keys(GENRES)` and list of labels (for rendering) `Objest.values(GENRES)`...